src/users/components/invite-user-permissions/invite-user-permission.component.ts

```diff
@@ -0,0 +1,15 @@
+import {Component, Input, Output, EventEmitter} from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+@Component({
+    selector: 'app-invite-permission-form',
```

Comment:
Not the end of the world, but maybe the selector should be `app-invite-user-permission`